src/v2/Apps/Artist/Routes/Overview/Components/ArtistRelatedArtistsRail.tsx

```diff
@@ -45,67 +45,66 @@ const ArtistRelatedArtistsRail: React.FC<ArtistRelatedArtistsRailProps> = ({
           }
 
           return (
-            <RouterLink
-              to={node.href!}
-              key={index}
-              noUnderline
-              onClick={() => {
-                tracking.trackEvent({
-                  action_type: AnalyticsSchema.ActionType.Click,
-                  contextModule: ContextModule.relatedArtistsRail,
-                  contextPageOwnerId,
-                  contextPageOwnerSlug,
-                  contextPageOwnerType,
-                  destination_path: node.href,
-                  destinationPageOwnerId: node.internalID,
-                  destinationPageOwnerSlug: node.slug,
-                  destinationPageOwnerType: OwnerType.artwork,
-                  horizontalSlidePosition: index + 1,
-                  type: "thumbnail",
-                })
-              }}
-            >
-              <Box>
+            <React.Fragment key={index}>
```

Comment:
why move key to a fragment instead of keeping it on RouterLink?

Reply:
its the outer container now. with the entity header this had invalid dom nesting